unit/constant/generate_constants.go

```diff
@@ -0,0 +1,327 @@
+// Copyright ©2014 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
+import (
+	"bytes"
+	"go/format"
+	"log"
+	"math"
+	"os"
+	"strings"
+	"text/template"
+
+	"gonum.org/v1/gonum/unit"
+)
+
+const (
+	elementaryCharge = 1.6021766208e-19
```

Comment:
For instance, in about 2 months this value will be defined to be 1.602176634e-19